index.d.ts

```diff
@@ -0,0 +1,7 @@
+import { BackendModule } from "i18next";
+
+export function mainBindings(ipcMain: Electron.IpcMain, browserWindow: Electron.BrowserWindow, fs: any): any;
+export function clearMainBindings(ipcMain: Electron.IpcMain): any;
+export function preloadBindings(ipcRenderer: Electron.IpcRenderer, process: NodeJS.Process): any;
+
+export default {} as BackendModule;
```

Comment:
Can you help me understand this line - why are we choosing "BackendModule" as the name of the default export after it's been imported from the i18next package? Can you give a little detail of your changes here, I'm sure this is something simple that I am not seeing.

Reply:
oooops im really sry that didn't answered, was very busy.
BackendModule choosed because its type from `i18next` that used for i18next backend modules like yours
![image](https://user-images.githubusercontent.com/14001879/181067097-279de15c-4081-4269-934d-3e230fcf55b4.png)

Reply:
Thanks @Ciberusps. I also mean to ask _what_ does `export default {} as BackendModule;` do? I'm a little confused about the `export default {}` part, why are we exporting an empty object. I'm not as familiar with Typescript syntax.

Reply:
So `.d.ts` files only describe what `.js` files do, empty object here required to add type `BackendModule`. 
Also `.d.ts` files not used in realtime, they only required for development.

tbh probably here it was better for understanding to add named export
![image](https://user-images.githubusercontent.com/14001879/181571951-7cd074bf-fc85-476b-8046-a3f486747e35.png)